pressbooks/pressbooks
# Fix positioning for images without caption in PDFs

This should fix #1741 and fix #2157

This PR attempts to solve PDF and EPUB floating image positioning.
It changes a bit the way RegEx was replacing content and tries to cover some edge cases.

I added some assertions to validate that an image is being wrapped by a `div.wp-nocaption`.

#### How to test

1. Create a book with floating (left, right) and centered images
2. Check that the book renders properly as web pages
3. Export a PDF and/or EPUB version of the book and check if the content is properly rendered
